charts/tempo-distributed/values.yaml

```diff
@@ -850,8 +855,8 @@ gateway:
   # -- Enable logging of 2xx and 3xx HTTP requests
   verboseLogging: true
   image:
-    # -- The Docker registry for the gateway image
-    registry: docker.io
+    # -- The Docker registry for the gateway image. Overrides `global.image.registry`
+    registry: null
```

Comment:
Yes, but the global default is [null](https://github.com/grafana/helm-charts/blob/main/charts/tempo-distributed/values.yaml#L4). based on the [tempoImage](https://github.com/grafana/helm-charts/blob/main/charts/tempo-distributed/templates/_helpers.tpl#L31) template, this will fall back on the tempo.image.registry default value which has been set with `docker.io`

I changed it to be consistent with the other image settings.

Reply:
understood. thx!